src/app/components/student-round/student-round.component.html

```diff
@@ -4,14 +4,25 @@
 
   <ng-container *ngSwitchDefault>
     <app-hill-background cover></app-hill-background>
-    <div class="centered-container" [ngSwitch]="currentScreen$ | async">
+    <div class="centered-container centered-text" [ngSwitch]="currentScreen$ | async">
 
       <mat-card *ngSwitchCase="ScreenId.Paused">
         <mat-card-title>Paused</mat-card-title>
       </mat-card>
 
       <mat-card *ngSwitchCase="ScreenId.InactiveBee">
-        <mat-card-title>Your {{(roundService.currentBeeSpecies$ | async).name}} is currently inactive</mat-card-title>
+        <img
+          class="student-bee"
+          *ngIf="(roundService.currentBeeSpecies$ | async) as bee"
+          src="/assets/art/500w/bees/{{bee.art_file}}"
+          [alt]="bee.name"
+        >
+        <mat-card-title>
+          {{(roundService.currentBeeSpecies$ | async).name | sentenceCase}} are currently inactive
+        </mat-card-title>
+        <mat-card-subtitle *ngIf="(nextActivePeriodText$| async) as nextActivePeriodText">
+          You will next be active in {{nextActivePeriodText}}
```

Comment:
```suggestion
          You will next be active {{nextActivePeriodText}}
```

The "in" is included in this string where needed. We don't want it to say "in soon" for example.